kubernetes-deployment/bootstrap.py

```diff
@@ -436,6 +480,7 @@ def main():
     parser.add_argument('-f', '--file', default=None, help="An yamlfile with the nodelist to maintain")
     parser.add_argument('-a', '--add', action="store_true", help="Add the node from nodelist.yaml")
     parser.add_argument('-r', '--remove', action="store_true", help="Remove the node from nodelist.yaml")
+    parser.add_argument('-m', '--maintain', default=None, help="maintain the cluster")
```

Comment:
>add_argument [](start = 11, length = 12)

this is getting more and more complicated. we need a printUsage to help user.

Reply:
For example?